debugger/src/dconfig.c

```diff
@@ -269,8 +274,13 @@ static void save_to_keyfile(GKeyFile *keyfile)
  */
 static gpointer saving_thread_func(gpointer data)
 {
+#if GTK_CHECK_VERSION(3, 0, 0)
+	gint64 interval;
+	g_mutex_lock(&change_config_mutex);
+#else
```

Comment:
I assume this code could be used for all gtk versions, again, not gtk but glib dependent.

Reply:
I agree. I replaced the deprecated glib code with the new one.